__tests__/client/prerender.spec.js

```diff
@@ -175,3 +179,55 @@ describe('moveHelmetScripts', () => {
     expect(NodeList.prototype.forEach).not.toHaveBeenCalled();
   });
 });
+
+describe('loadPWA', () => {
+  let error;
+
+  beforeAll(() => {
+    ({ error } = console);
+    console.error = jest.fn();
```

Comment:
instead of overwriting, use a spy. if the purpose is to prevent the logs from showing up in the tests then use `mockImplementation` on the spy

Reply:
I have updated.